app/src/styles.sass

```diff
@@ -164,15 +169,20 @@ $itero-mat-theme: mat-light-theme(( color: ( primary: $itero-mat-primary, accent
   right: 0
   display: flex
   flex-flow: row nowrap
+  transform: translate(85%,0)
```

Comment:
This is not clear whether to close position is with 85% or 90%, because the style is spread between this file and the TypeScript code. Anyway, the space on the right of the icon seems inconsistent to me.

320px | 414px | 500px
--------|----------|---------
![image](https://user-images.githubusercontent.com/1835665/122024645-ea1f3880-cdc8-11eb-90b6-88d6aeb80010.png) | ![image](https://user-images.githubusercontent.com/1835665/122025072-5306b080-cdc9-11eb-8372-afeb59c5f3e5.png) | ![image](https://user-images.githubusercontent.com/1835665/122025161-6ca7f800-cdc9-11eb-9dcf-2a6ce54f775d.png)